src/index.js

```diff
@@ -48,7 +49,10 @@ const SDK = {
   AuthorizationGrant: User.AuthorizationGrant,
 
   // Ping
-  ping
```

Comment:
Good question. This is because the API that is documented and what we had been doing is `Kinvey.ping()` vs the other stuff is `Kinvey.User.login()` etc.